website/docs/support/bugs-and-features.md

```diff
@@ -45,3 +45,151 @@ Next, file a [GitHub issue](https://github.com/alcionai/corso/issues/) with the
 ([default log file](../../setup/configuration/#log-files) and `corso-gc.log`, the Corso GC log file, from above) and
 information on the size of the Exchange mailbox, OneDrive location, or SharePoint site that you are having an issue
 with.
+
+## Sharepoint List Anomalies
+
+Here is an example of GRPAH API response of columns of a list:
+
+```json
```

Comment:
This seems quite verbose IMO. Can we move details to a ticket and link the issue here?